### ISSUE TITLE
Slack join link no longer works

### ISSUE THREAD

**Issue by thibaudcolas** [opencollective/opencollective#7193]
When I try to access <https://slack.opencollective.com/>, it redirects to a Slack join link, and I get the following error:

> This link is no longer active
> To join this workspace, you’ll need to ask the person who originally invited you for a new link.

If it helps, the join link is:

```
https://join.slack.com/t/opencollective/shared_invite/zt-24f99sib5-e_WpIuOjYk03SxM_~7CyPA
```

**Comment by Betree** [comment]
@thibaudcolas Thank you for reporting this; we have renewed the link. The root cause is documented in https://github.com/opencollective/opencollective/issues/7023.

**Comment by thibaudcolas** [comment]
Thank you @Betree :) We have the similar problem with other orgs I’m involved with, I’ll see if I can figure out how we solved it and report back in #7023.
